src/ir/cost.h

```diff
@@ -789,13 +460,22 @@ struct CostAnalyzer : public Visitor<CostAnalyzer, Index> {
     return ret + visit(curr->left) + visit(curr->right);
   }
   Index visitSelect(Select* curr) {
-    return 2 + visit(curr->condition) + visit(curr->ifTrue) +
+    return 1 + visit(curr->condition) + visit(curr->ifTrue) +
            visit(curr->ifFalse);
   }
```

Comment:
@kripken I think `cmov` and `jmp` should have same costs, so I switch base value to `1` similar to `visitIf`

Reply:
Sounds reasonable.